src/ROUTES.ts

```diff
@@ -612,6 +612,23 @@ const ROUTES = {
         route: 'settings/workspaces/:policyID/members/:accountID/role-selection',
         getRoute: (policyID: string, accountID: number, backTo?: string) => getUrlWithBackToParam(`settings/workspaces/${policyID}/members/${accountID}/role-selection`, backTo),
     },
+    WORKSPACE_OWNER_PAYMENT_CARD_FORM: {
+        route: 'settings/workspaces/:policyID/add-payment-card/:accountID',
```

Comment:
I'm thinking that we should just make this route be part of the change checks, e.g.  `settings/workspaces/:policyID/change-owner/:accountID/:error` and in this case error would be `add-payment-card`. This would avoid the transition that happens from loading indicator to form page.

https://github.com/Expensify/App/assets/22219519/7eec6076-cf9a-41d9-9566-5abb31e47ab4

Reply:
yeah, makes sense - unified these two pages into one